### PR TITLE
handle the stop codons, COSMIC errors download, ENSEMBL releases 

### PR DESCRIPTION
**Pull request by ypriverol**
- [x] handle the stop codons in ensembl databases 
- [x] add new test for stop codons.  
- [x] Cosmic error handling when the user can't log in. 
- [x] Ensembl release issue #23 
- [x] Add changelog.md to the project
- [x] Add code quality service.

### CODE REVIEW

**Comment by ypriverol** [comment]
@husensofteng do not merge yet the PR, still fixing COSMIC login error

**Comment by ypriverol** [comment]
@husensofteng the ensembl grch37 is implemented. Please Review the present PR.

**Comment by ypriverol** [comment]
@husensofteng the PR is now to be merge.

File: pypgatk/commands/dnaseq_to_proteindb.py [hunk 1]
```diff
@@ -9,8 +9,8 @@
 
 
 @click.command("dnaseq-to-proteindb", short_help="Generate peptides based on DNA sequences")
-@click.option('--config_file', '-c', help='Configuration to perform conversion between ENSEMBL Files',
-              default= this_dir + '../config/ensembl_config.yaml')
+@click.option('-c', '--config_file', help='Configuration to perform conversion between ENSEMBL Files',
+              default= this_dir + '/../config/ensembl_config.yaml')
```
Comment (husensofteng) [inline pypgatk/commands/dnaseq_to_proteindb.py:13]: using `this_dir` makes issues when the default is used. why not just the default config file without `this_dir`?

File: pypgatk/commands/threeframe_translation.py [hunk 2]
```diff
@@ -10,10 +10,9 @@
 
 
 @click.command('threeframe-translation', short_help="Command to perform 3'frame translation")
-@click.option('--config_file',
-              '-c',
+@click.option('-c', '--config_file',
               help='Configuration to perform conversion between ENSEMBL Files',
-              default= this_dir + '../config/ensembl_config.yaml')
+              default= this_dir + '/../config/ensembl_config.yaml')
```
Comment (husensofteng) [inline pypgatk/commands/threeframe_translation.py:15]: this_dir introduces error when the default is used.

File: pypgatk/ensembl/data_downloader.py [hunk 3]
```diff
@@ -16,326 +16,367 @@
...
+    try:
+      if grch37:
+        species['assembly'] = 'GRCh37'
+      file_name = '{}.{}.cds.all.fa.gz'.format(species['name'][0].upper() + species['name'][1:],
+                                               species['assembly'])
+      file_url = '{}/release-{}/fasta/{}/cds/{}'.format(
+        self.get_default_parameters()[self.CONFIG_KEY_DATA_DOWNLOADER][self.CONFIG_KEY_ENSEMBL_FTP][
+          self.CONFIG_KEY_BASE_URL], species['release'], species['name'], file_name)
+      if grch37:
+        file_url = '{}/grch37/release-{}/fasta/{}/cds/{}'.format(
```
Comment (husensofteng) [inline pypgatk/ensembl/data_downloader.py:215]: release number should be a user parameter, if not given then get it from species['release']

**Review by husensofteng — APPROVED** [review]

File: pypgatk/ensembl/data_downloader.py (hunk 3 above)
Comment (ypriverol) [reply pypgatk/ensembl/data_downloader.py:215]: Would be good to have the releases previous to 38, the problem is that ENSEMBL do not give that information with the same structure. Lets wait until a user ask for the feature in the future.

File: pypgatk/commands/threeframe_translation.py (hunk 2 above)
Comment (ypriverol) [reply pypgatk/commands/threeframe_translation.py:14]: how are you running it, for me is working from the package.

**Pull request merged**